sources/views/contacts/formInContacts.js

```diff
@@ -1,21 +1,50 @@
 import {JetView} from "webix-jet";
+import { contacts } from "models/contacts.js";
+import { countries } from "models/countries";
+import { statuses } from "models/statuses";
 
 export default class FormInContactsView extends JetView {
 	config() {
+		const _ = this.app.getService("locale")._;
 		return {
 			view: "form",
+			localId: "myForm",
 			autoheight: false,
 			elements: [
-				{view: "text", label: "Name"},
-				{view: "text", label: "E-mail"},
+				{view: "text", label: _("Name"), name: "Name"},
+				{view: "text", label: _("E-mail"), name: "Email"},
+				{view: "combo", label: _("Countries"), name: "Country", value:"",
+					options:{body: {template:"#Name#"}, data: countries}},
+				{view: "combo", label: _("Statuses"), name: "Status", value:"",
+					options:{body: {template:"#Name#"}, data: statuses}},
 				{
 					cols: [
-						{view: "button", value: "Add"},
-						{view: "button", value: "Clear"}
+						{view: "button", value: _("Save"), click: () => this.saveContact()}
 					]
 				}
-
 			]
 		};
 	}
+
+	init(){
+		this.form = this.$$("myForm");
+	}
+	urlChange(view, url) {
+		const idFromUrl = url[0].params.id;
+		if(contacts.config.data.length){
```

Comment:
you need to check it as `.exists()`  - it covers both cases when contact does not exist and there are no contacts